libtcmu_common.h

```diff
@@ -75,6 +77,9 @@ int tcmu_emulate_read_capacity_16(uint64_t num_lbas, uint32_t block_size, uint8_
 int tcmu_emulate_mode_sense(uint8_t *cdb, struct iovec *iovec, size_t iov_cnt, uint8_t *sense);
 int tcmu_emulate_mode_select(uint8_t *cdb, struct iovec *iovec, size_t iov_cnt, uint8_t *sense);
 
+void tcmulib_track_aio_request_start(struct tcmu_device *dev);
+void tcmulib_track_aio_request_finish(struct tcmu_device *dev, int *is_idle);
```

Comment:
Why export these in the header? I do not think they are useful without being able to use aio_schedule right? Is it part of the rbd aio patches?